### ISSUE TITLE
move foodsharing.de from operators/amenity/social_facility to brands/amenity/food_sharing

### ISSUE THREAD

**Issue by tiptoptom** [osmlab/name-suggestion-index#10226]
[This](https://github.com/osmlab/name-suggestion-index/blob/5759a99c8c28b20d1128ddf2ba7776956ad17313/data/operators/amenity/social_facility.json#L743C1-L755C7) preset is [not contained in the OSM data](https://overpass-turbo.eu/s/1VIs). It is also not clear to me what it is actually intended for. I think a preset in brands/amenity/food_sharing is better suited for the foodsharing.de "Fairteiler".

I suggest deleting the preset in operators/amenity/social_facility and creating one in brands/amenity/food_sharing for “ Fairteiler".

**Comment by Snowysauce** [comment]
[These are the results](https://overpass-turbo.eu/s/1YSz) for searching for `operator=foodsharing.de` and `amenity=social_facility`. Thus, there is OSM data that matches the preset, even if the POIs don't carry the Wikidata tag for the operator. Whether the POIs _should_ match the preset is another question entirely.

**Comment by tiptoptom** [comment]
> [These are the results](https://overpass-turbo.eu/s/1YSz) for searching for `operator=foodsharing.de` and `amenity=social_facility`.

There are 13 of them worldwide, almost all in the same city. I wouldn't be impressed by that.

**Comment by Snowysauce** [comment]
It may not be impressive, but I tested removing the preset from the category file, and the build script re-adds it from the OSM planet data. The preset would have to be explicitly excluded in the category file header, and since there is live OSM data that matches the preset tags, we'd have to have a specific reason to do so.

**Comment by tiptoptom** [comment]
For most of these nodes, it seems very clear to me that it is not a food bank where food is given to the needy but food sharing, where people can drop of and pick up food.

Should I just change the tagging for these nodes?

**Comment by Snowysauce** [comment]
> Should I just change the tagging for these nodes?

Yep, changing how the nodes are tagged in OSM is truly the only way to eliminate the `operators/amenity/social_facility` preset. (It also takes a fresh run of the OSM planet data collector, but that's on our end.)

**Comment by tiptoptom** [comment]
Do all nodes with operator=foodsharing.de and amenity=social_facility have to disappear for the data collector to stop hitting?

**Comment by Snowysauce** [comment]
Not all, but based on what I've seen, the number of instances likely has to be under 10 to avoid automatically created operator presets. (I'm not sure why the threshold seems to be just 10 instances for operators while automatic brand presets start at 50 instances, but this behavior seems to line up with what I saw while doing `locationSet` filtering on automated presets a while back.)

**Comment by tiptoptom** [comment]
There are now 8 nodes left.

**Comment by Snowysauce** [comment]
The ideal resolution for this issue depends on #10621 being completed; per #10536, that may not be for a while, if ever. Apparently the collection script requires a specific environment (OS, distros, etc.) and a higher level of code knowledge than that needed for the NSI itself, and there's a real possibility that the script may never be run again.

If the preset is truly undesirable, the second option mentioned in https://github.com/osmlab/name-suggestion-index/issues/10226#issuecomment-2663821790 may be the only option.

**Comment by Cj-Malone** [comment]
> It may not be impressive, but I tested removing the preset from the category file, and the build script re-adds it from the OSM planet data. The preset would have to be explicitly excluded in the category file header, and since there is live OSM data that matches the preset tags, we'd have to have a specific reason to do so.

Not necessarily, if OSM has 100 entries in category X, but NSI has an entry in category Y, If Y is it a match group with X, or the entry mentions match tags X, it won't be auto added to X.json.

**Comment by tiptoptom** [comment]
So what needs to be done @Cj-Malone ?

**Comment by Cj-Malone** [comment]
I have no idea what foodshareing.de is, but if you are confidant that this is right, and wont flip flop:

`"matchTags": ["amenity/social_facility"]`

**Comment by Snowysauce** [comment]
My understanding of `matchTags` is that, aside from the category file remapping, OSM editors use the value to suggest retagging entities in the matched category. (In other words, `"matchTags": ["amenity/social_facility"]` on an entry in another category would tell iD et al to suggest re-tagging everything tagged as `amenity=social_facility` with the new category's tag.) If everything that's currently tagged as `amenity=social_facility` fits into the new category, this approach should be fine; otherwise, we're sending bad suggestions to OSM editors that can lead to bad data in OSM.

**Comment by tiptoptom** [comment]
No, `amenity=social_facility` should not be replaced everywhere by `amenity=food_sharing`. I suspect `matchTags` are used as an or-condition. I think we would need an and-condition for this case like `amenity=social_facility and operator=foodsharing.de`.

**Comment by tiptoptom** [comment]
For my understanding we can just wait for a [new run of the nsi-collector](https://github.com/osmlab/name-suggestion-index/issues/10621#issuecomment-2699635741) and then remove `"matchTags": ["amenity/social_facility"]` again.

**Comment by Cj-Malone** [comment]
> I tested removing the preset from the category file, and the build script re-adds it from the OSM planet data. The preset would have to be explicitly excluded in the category file header, and since there is live OSM data that matches the preset tags, we'd have to have a specific reason to do so.

The entry gets added without wikidata, without wikidata it wont be recommanded by iD. You could add a note for it to be removed in the future, but also adding it to the excludes is OK. It's there to be used.

**Comment by Snowysauce** [comment]
Added to the exclusion list for two reasons: 1) to be frank, I'm tired of hearing about this issue and 2) even once the collector data is updated, I don't feel like going through the same song and dance at a later point whenever the `operator=foodsharing.de` usage threshold for automatic addition is inevitably reached again.
